src/Base64.cpp

```diff
@@ -97,4 +102,21 @@ string encodeBase64Url(const Data& val) {
     return encoded;
 }
 
+std::string encode_rust(const Data& val) {
```

Comment:
what's the difference of new added base64 encoding?

Reply:
It's seems that boost base64 iterator is not compliant, (see the unit tests for decode sui in Base64Tests.cpp - this one is compliant.

Reply:
in this case, maybe worth to remove using boost base64